.github/workflows/tag.yml

```diff
@@ -0,0 +1,37 @@
+name: Create Tag
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag name, defaults to minor increment."
+        required: false
+      sha:
+        description: "Commit SHA to tag. Defaults to latest."
+        required: false
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set the commit SHA
+        id: sha
+        run: echo "::set-output name=commitSha::$(if [ -z "$SHA" ]; then echo $GITHUB_SHA; else echo $SHA; fi)"
+        env:
+          SHA: ${{ github.event.inputs.sha }}
+      - name: Set the tag
+        id: tag
+        run: |
+          last_tag="$(git describe --tags --abbrev=0)"
+          new_tag="$(echo "$last_tag" | perl -pe 's/^(v\d+\.)(\d+)(\.\d+)$/$1.($2+1).($3)/e')"
```

Comment:
This will keep the existing patch version which we probably don't want if bumping up the minor, right?